Revit_Engine/Query/CategoryNames.cs

```diff
@@ -41,7 +41,7 @@ public static partial class Query
         [Output("CategoryNames")]
         public static List<string> CategoryNames(this FamilyLibrary familyLibrary, string familyName, string familyTypeName = null)
         {
-            if (familyLibrary == null)
+            if (familyLibrary == null || familyLibrary.Dictionary == null)
```

Comment:
@pawelbaran am I missing something, but I can't see where this provides `Informative error given when requested family type does not exist in the Revit document`? I am probably missing something so just a comment, not a review :smile:

Reply:
Oh how can you not see it 🙈 The warning about lack of matching type is triggered by some other part of the code that comes after that method call - it was already in place and seems to be working fine. Good catch though ;-)

Reply:
Fair enough @pawelbaran - just thought I'd check :smile: All good here!